war/src/main/webapp/css/style.css

```diff
@@ -1,18 +1,18 @@
 /*
  * The MIT License
- * 
+ *
```

Comment:
@ssbarnea My suggestion is to avoid this kind of style changes.

Reply:
@recena style change or repairing previous errors? My editor (atom) remove training spaces by default, which is a good practice anyway. Sadly Jenkins code linting is very poor.

Yes, i can fix it but I think it should not be needed.